src/aave-v3/SupplyVault.sol

```diff
@@ -18,6 +19,7 @@ contract SupplyVault is ISupplyVault, SupplyVaultBase {
     using FixedPointMathLib for uint256;
```

Comment:
Perhaps we can even remove this part no?

Reply:
Will do if the discussion below agrees on using wadraymath.